src/ImagineSvg/Image.php

```diff
@@ -0,0 +1,461 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\ImagineSvg;
```

Comment:
Aah, so these are actually our SVG drivers for imagine? They don't belong in the bundle at all imho, they're a library. But that's a difficult topic :D